lib/routes/account.js

```diff
@@ -946,10 +1013,33 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+
+          var doSigninConfirmation = requestHelper.shouldEnableSigninConfirmation(sessionToken.uid.toString('hex'), config)
+
+          if (doSigninConfirmation) {
+            return db.sessionTokenWithVerificationStatus(sessionToken.id)
```

Comment:
> But it's not clear to me what routes would use a strict `sessionTokenVerified` strategy.

Oh, excellent point. I think the use of `makeVerifiedCredentialsFn` is just hangover from before we shifted the responsibility for rejection into the token server. It should be fine to just fix `sessionTokenWithVerificationStatus` I think.